### PR TITLE
fix(server): reinitialize the progress to set up graph auth friendly

### PR DESCRIPTION
**Pull request by Z-HUANT**
## Purpose of the PR

修复需要重新初始化数据库才能设置 auth 的问题
fixed https://github.com/apache/incubator-hugegraph/issues/125#issuecomment-717713131

## Main Changes

Switch to auth-mode without removing graph data (不需要重新初始化数据库就可以设置 auth)

## Verifying these changes

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [x] Need tests and can be verified as follows:

 自己测试流程如下
1. 使用非 auth init-store 初始化图，此时无 admin 账号
2. 使用非 auth 启动 HugeGraph，正常添加和查询数据（模拟用户操作）
3. 使用 auth init-store 初始化图，创建 admin 账号，模拟用户运行一段时间后，数据库内有数据了后，想增加 auth 
4. 使用 auth 启动 HugeGraph，可以正常启动，权限校验正常，此时访问接口都需要账号密码
5. 使用非 auth 启动 HugeGraph，模拟用户不想要 auth 了，此时可以正常启动，访问接口不需要账号密码

## Does this PR potentially affect the following parts?

- [ ]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [x]  Other affects (typed here)

影响 init-store

## Documentation Status

- [ ]  `Doc - TODO` 
- [x]  `Doc - Done` 
- [ ]  `Doc - No Need` 

Doc：https://github.com/apache/incubator-hugegraph-doc/pull/320

### CODE REVIEW

File: hugegraph-server/hugegraph-dist/src/main/java/org/apache/hugegraph/cmd/InitStore.java [hunk 1]
```diff
@@ -101,6 +101,8 @@ private static HugeGraph initGraph(String configPath) throws Exception {
                 LOG.info("Skip init-store due to the backend store of '{}' " +
                          "had been initialized", graph.name());
                 backendStoreInfo.checkVersion();
+                // Initializing the information required for creating the admin account.
```
Comment (imbajin) [inline hugegraph-server/hugegraph-dist/src/main/java/org/apache/hugegraph/cmd/InitStore.java:104]:
```suggestion
                // Init the required information for creating the admin account (when switch from non-auth mode to auth)
```

File: hugegraph-server/hugegraph-dist/src/main/java/org/apache/hugegraph/cmd/InitStore.java [hunk 2]
```diff
@@ -101,6 +101,8 @@ private static HugeGraph initGraph(String configPath) throws Exception {
                 LOG.info("Skip init-store due to the backend store of '{}' " +
                          "had been initialized", graph.name());
                 backendStoreInfo.checkVersion();
+                // Init the required information for creating the admin account (when switch from non-auth mode to auth mode)
```
Comment (javeme) [inline hugegraph-server/hugegraph-dist/src/main/java/org/apache/hugegraph/cmd/InitStore.java:104]: please wrap line if exceed 80~100 chars

**Review by javeme — APPROVED** [review]
LGTM

**Review by imbajin — APPROVED** [review]

File: hugegraph-server/hugegraph-dist/src/main/java/org/apache/hugegraph/cmd/InitStore.java [hunk 3]
```diff
@@ -101,6 +101,11 @@ private static HugeGraph initGraph(String configPath) throws Exception {
                 LOG.info("Skip init-store due to the backend store of '{}' " +
                          "had been initialized", graph.name());
```
Comment (imbajin) [inline hugegraph-server/hugegraph-dist/src/main/java/org/apache/hugegraph/cmd/InitStore.java:102]: @Z-HUANT seems we could move this log after L108? (due to the code logic)

could enhance it in another reviewed PR

Comment (Z-HUANT) [reply hugegraph-server/hugegraph-dist/src/main/java/org/apache/hugegraph/cmd/InitStore.java:102]: I'll put it in this PR later: https://github.com/apache/incubator-hugegraph/pull/2408.

**Pull request merged**